home.html

```diff
@@ -1,43 +1,93 @@
-                <h2>Our planet is precious and vital for our
-                    beautiful lives, if we don’t take care of it it
-                    will dissapear along with us.</h2>
-                    <h4>*Please take note that your actions matter more than you think.</h4>
-                    <a href="#">Find out more</a>
-                    <a href="#">Play video</a>
+<body class="main">
+  <section class="main_section">
+    <div class="side">
+      <p class="side_text">+04 070 991 598 * wild@life.com * Wild Avenue, San Francisco, CA</p>
+
+      <div class="hero">
+        <div class="logo">
+          <img src="css/logo.png" alt="" class="logo_hero">
+        </div>
+        <div class="main_text">
+          <h5 class="required">We teach you to protect our</h5>
+          <p class="banner">PLANET</p>
+          <h2 class="main_text__hero">Our planet is precious and vital for our
+            beautiful lives, if we don’t take care of it it
+            will dissapear along with us.</h2>
+          <h4 class="main_text__note">*Please take note that your actions matter more than you think.</h4>
+          <a href="#" class="main_button__left">Find out more</a>
+          <a href="#" class="main_button__right">Play video</a>
+        </div>
+        <div class="nav_area">
+          <ul class="nav">
+            <li class="nav_item">
+              <a href="#" class="nav_list__item">Home</a>
+            </li>
+            <li class="nav_item">
+              <a href="#" class="nav_list__item">Story </a>
+              <div class="dropdown">
+                <ul>
+                  <li class="dropdown_list"><a href="#" class="dropdown_list__item">Team</a></li>
+                  <li class="dropdown_list"><a href="#" class="dropdown_list__item">Project</a></li>
+                  <li class="dropdown_list"><a href="#" class="dropdown_list__item">Future</a></li>
+                  <div class="arrow"></div>
+                </ul>
               </div>
+            </li>
+            <li class="nav_item">
+              <a href="#" class="nav_list__item">Contact</a>
+            </li>
+            <li class="nav_item">
+              <a href="#" class="nav_list__item">Lessons</a>
+            </li>
+            <li class="nav_item">
+              <a href="#" class="nav_list__item">Account</a>
+            </li>
+          </ul>
+        </div>
 
-              <ul class="nav-area">
-              <li><a href="#">Home</a></li>
-              <li><a href="#">Story</a></li>
-              <li><a href="#">Contact</a></li>
-              <li><a href="#">Lessons</a></li>
-              <li><a href="#">Account</a></li>
-              </ul>
+      </div>
+    </div>
+  </section>
+
+  <div class="gallery">
+    <p class="gallery_title">Our Mission</p>
+  </div>
+  <div class="gallery_paragraph">
+    <p class="gallery_text">Our wildlife is important to us<br>if we don’t take care of it<br>maybe one day will turn<br>against us and we will die.</p>
```

Comment:
nu folosi br-uri in cazul asta, pe responsive o sa se strice rau de tot. ca sa treaca la urmatorul rand te folosesti de padding-right in cazul asta, pana cand are acelasi width ca sa treaca la urmatorul rand cand trebuie. Astfel la responsive o sa poti ajusta padding-ul. In cazul cu br, nu le vei putea scoate pe responsive.